setup.py

```diff
@@ -111,6 +111,37 @@ def get_packages():
     packages = get_subpackages(LIBNAME)
     return packages
 
+def get_qt_requirements(qt_requirements, default='pyqt5'):
+    """
+    Return a list of requirements for the Qt binding according to the
+    environment variable SPYDER_QT_BINDING. If this variable is not set
+    or has an unsupported value it defaults to 'pyqt5'.
```

Comment:
Just a nitpick: In my opinion it’s better to throw an error when an invalid/unsupported value is passed.

Reply:
Raising ValueError when it couldn't find.